sdks/jreleaser-git-java-sdk/src/main/java/org/jreleaser/sdk/git/ChangelogGenerator.java

```diff
@@ -608,6 +612,11 @@ Map<String, Object> asContext(boolean links, String commitsUrl) {
             context.put("commitTitle", passThrough(title));
             context.put("commitAuthor", passThrough(author.name));
             context.put("commitBody", passThrough(body));
+            context.put("commitIssuesCount", issues.size() > 0 ? issues.size() : null);
```

Comment:
I've used a boolean in the past to  mark that a collection may be empty. Setting `null` for a count might not be a good idea. Setting `0` would better unless Mustache's thruthy value takes `0` as true.

Reply:
I tried 0 and it doesn't work. I can use a boolean if you prefer.

Reply:
Yes, I think the boolean will make it consistent. The name should be `commitHasIssues`.